examples/certifier_kafka_pg/examples/certifier_kafka_pg.rs

```diff
@@ -18,7 +18,21 @@ async fn main() -> Result<(), impl std::error::Error> {
 
     info!("Talos certifier starting...");
 
-    let kafka_config = KafkaConfig::from_env(None);
+    let mut kafka_config = KafkaConfig::from_env(None);
+    kafka_config.extend(
+        None,
+        Some(
+            [
+                ("enable.auto.commit".to_string(), "false".to_string()),
+                ("auto.offset.reset".to_string(), "earliest".to_string()),
```

Comment:
Is this push intentional?

Reply:
yes that is intentional, as I need that till I enable the manual commit. This helps me in consuming the same set of messages from Kafka and run through my new changes in code.

Reply:
oh sorry, just realised that in the `certifier`, I think this is part of the commit before your PR to fix the auto commit went it. I will remove this now.